MewsSystems/mews-flutter
# feat(optimus): [RND-103479] Add alignment prop to typography components

#### Summary

- Added missing `align` property to the typography components. By default, it will resolve to the `TextAlign.left`, so it should not bring any visual changes as previously it was set to `TextAlign.start` anyway.
- Updated stories

<details><summary>Screenshots</summary>

`OptimusTypographyAlignment.left`

![CleanShot 2022-08-02 at 07 26 51@2x](https://user-images.githubusercontent.com/9210422/182298529-99548084-8d76-498d-a31f-a2740cca2cea.png)

`OptimusTypographyAlignment.right`

![CleanShot 2022-08-02 at 07 26 58@2x](https://user-images.githubusercontent.com/9210422/182298534-1c02a401-be11-407b-b1bf-17cf0bc68704.png)

`OptimusTypographyAlignment.center`

![CleanShot 2022-08-02 at 07 27 09@2x](https://user-images.githubusercontent.com/9210422/182298540-4198577e-90fd-44dc-8e47-8ef794c9bb7b.png)


</details> 

#### Testing steps

1. Open storybook
2. Open any typography coponent
3. Change it to very long text
4. Change the `align` property and check if it is displaying correctly.

#### Follow-up issues

None.

#### Check during review

- Verify against YouTrack issue.
- Is the PR over 300 additions? Consider rejecting it with advice to split it. Is it over 500 additions? It should definitely be rejected.
- Unused code removed.
- Build passing.
- Is it a bug fix? Check that it is covered by a proper test (unit or integration).
